ape_starknet/ecosystems.py

```diff
@@ -450,3 +450,6 @@ def _get_proxy_info(
 
     def decode_primitive_value(self, value: Any, output_type: Union[str, Tuple, List]) -> int:
         return to_int(value)
+
+    def decode_calldata(self):
+        raise NotImplementedError
```

Comment:
```suggestion
        raise NotImplementedError()
```

We may want to try and implement this one if we have time.